db/migrate/20230920110724_enable_pg_crypto.rb

```diff
@@ -0,0 +1,5 @@
+class EnablePgCrypto < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension 'pgcrypto'
```

Comment:
I remember database portability being a concern for some of the NBU work in case we ever want to move to MySQL or something. Would this be easy to port over to the equivalent feature in a different DB in that case?

Reply:
This is definitely a valid point. I think that there's already some PostgreSQL dependency in this project with the query syntax (`column ILIKE ?`) and JSON support (`user_defined->key = ?`). Moving to a different database platform would likely require more changes than just what is included in this PR. At this point, it's probably okay to kick that problem down the road a bit and deal with it if/when we get there.